### ISSUE TITLE
Enable ctrl+click on new table rows

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#7100]
It would be really comfortable to open multiple tabs by ctrl+click on multiple rows in a table. 
Is it something we could do with the new table?

cc @GrandSchtroumpf

**Comment by GrandSchtroumpf** [comment]
The issue here is that a row of a table cannot be an anchor (`<a>` tag), so browsers don't know that this is a link.

There are multiple unsatisfying workaround : 
- Using a grid instead of a table, which would increase CSS complexity & reduce browser performance on managing the columns. Plus might not have the same effect on every browser.
- Adding a `a` tag on every cell of every table with a link. Which would be quite cumbersome to implement, prompt to error and difficult to maintain. Also it increases CSS complexity by adding a layer between row & text.
- Using a link as first or last column instead of having clickable row. This is quick easy to do, but reduces User Experience.

When I worked on the table I spent a day trying these solutions to use an anchor instead of a simple "click" event, but came back to the "click" event before the complexity on the long run or performance loss.
